src/parser/parsers/changelogs.py

```diff
@@ -11,27 +13,60 @@
 from .constants import OUTPUT_DIR
 
 
+
+class HTMLTagRemover(HTMLParser):
+    # https://www.slingacademy.com/article/python-ways-to-remove-html-tags-from-a-string#Using_HTMLParser
+    def __init__(self):
+        super().__init__()
+        self.result = []
+
+    def handle_data(self, data):
+        self.result.append(data)
+
+    def get_text(self):
+        return ''.join(self.result)
+
+
 class ChangelogParser:
     def __init__(self):
         self.CHANGELOGS_DIR = os.path.join(os.path.dirname(__file__), '../raw-changelogs/')
         self.OUTPUT_DIR = OUTPUT_DIR
-        self.OUTPUT_CHANGELOGS = self.OUTPUT_DIR+'/changelogs'
+        self.OUTPUT_CHANGELOGS = self.OUTPUT_DIR + '/changelogs'
         self.resources = self._get_resources()
 
-    def run_all(self):
-        changelogs_by_date = {}
+    # download rss feed from changelog forum and parse entries
+    def fetch_forum_changelogs(self):
+        print('Fetching Changelog RSS feed')
+        # Forum Link: https://forums.playdeadlock.com/forums/changelog.10/
+        f_url = 'https://forums.playdeadlock.com/forums/changelog.10/index.rss'
+        # fetches first page, last 20 entries date-wise
+        feed = feedparser.parse(f_url)
+        for entry in feed.entries:
+            # Dependent on thread title being in the format "mm-dd-yyyy Update"
+            date = entry.title.replace(' Update', '')
+            html_remover = HTMLTagRemover()
+            # Potential for more than one post, we just grab the first
+            html_remover.feed(entry.content[0].value)
+            self.changelogs_by_date[date] = self.run(date, html_remover.get_text())
```

Comment:
I would add a step to save the raw changelog along with the others, as its nice to keep our own archive

Reply:
Done!  I created a "raw-changelogs" folder in the output directory, and have rss and txt subfolders that contain the changelogs